events/2025-04-22-community-call-hub-cloud-costs/index.qmd

```diff
@@ -0,0 +1,53 @@
+---
+title: "Community Call: Beyond “it depends”: what we’re learning about cloud costs for Earth science workflows in our JupyterHub"
```

Comment:
```suggestion
title: "Community Call: What we’re learning about cloud costs for Earth science workflows in our JupyterHub"
```
As the title has evolved/improved, I think the "beyond it depends" might make it too long? I first suggested that thinking it was punchy, and am also not tied to it. What do you think Stef?